tame.lua

```diff
@@ -21,9 +21,14 @@ Usage: ``tame -read`` OR ``tame -set <level>``
 
 local utils = require('utils')
 local selected = dfhack.gui.getSelectedUnit()
-local validArgs = utils.invert({'set','read'})
+local validArgs = utils.invert({'set','read','help'})
 local args = utils.processArgs({...}, validArgs)
 
+if args.help then
+    print(help)
+    do return end
```

Comment:
Just `return` here would be enough - `do return end` is only necessary in rare cases where you have code *between* an `return` statement and `end`

Reply:
I know, just being safe; I copy paste code around a lot. Just avoiding high risk code from breaking if I move if for "reasons"